source/guides/core.rst

```diff
@@ -14,8 +14,15 @@ Development Process
    /process/release-process*
    /process/accepting-pull-request*
    /process/community-systems*
-   /process/documentation-guidelines*
+   /process/documentation-guidelines.md
 
+Documentation Style Guide
+=========================
+
+.. toctree::
+   :maxdepth: 3
+
+   /process/sg_mattermost-doc-style.rst
```

Comment:
Sorry if I missed this, just wondering if we should add a note on file naming convention? 

It seems this PR has `sg_` as a prefix, which is different from how other pages are organized. If we use a prefix either we're going to be inconsistent across our pages or we'll have to redirect or break links to refactor everything to use prefixes (which is not bad, so long as we know how to do that properly--what we don't want is orphaned pages). 

Our file naming isn't consistent yet, we should figure out if file naming is in scope for the style guide